File: Solutions by Programming Task/Fibonacci sequence/Visual Basic .NET/Iterative.vb
Imports System.Numerics
Function Fib(x As BigInteger) As BigInteger
    If x < 2 Then Return x
    Dim p As BigInteger = 0
    Dim n As BigInteger = 1
    For i = 0 To x - 2
        Dim s = p + n
        p = n
        n = s
    Next
    Return n
End Function

File: Solutions by Programming Task/Fibonacci sequence/Rust/Iterative.rs
fn fib(x: usize) -> usize {
    let mut d: (usize, usize) = (0, 1);
    for _ in 0..x {
        d = (d.1, d.0 + d.1)
    }
    d.0
}

fn main() {
    println!("{}", fib(10));
}
